### PR TITLE
fix: Disabled pinch to zoom on mobile devices#9155

### LINKED ISSUE #9155: [CAL-1799] prevent zoom on mobile usage
on mobile, disable pinch to zoom

<sub>[CAL-1799](https://linear.app/calcom/issue/CAL-1799/prevent-zoom-on-mobile-usage)</sub>

### PR DESCRIPTION
**Pull request by Gohan-18**
## What does this PR do?
Disables pinch to zoom in mobile devices

Fixes #9155  (issue)

**Environment**: Staging(main branch) / Production

## Type of change

#- Bug fix (non-breaking change which fixes an issue)
#- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
#- Breaking change (fix or feature that would cause existing functionality to not work as expected)
#- This change requires a documentation update

## How should this be tested?

- [x] Test A
- [x] Test B

### CODE REVIEW

**Comment by sean-brydon** [comment]
@PeerRich  what is the reason for this? Its generally recommended to avoid disabling pinch to zoom as its not the best thing to do for a11y.

**Comment by PeerRich** [comment]
@sean-brydon zooming on mobile apps looks weird and breaks the pwa experience. i dont mind keeping it, just looks weird. visit the mobile page and pinch to zoom, looks odd

**Review by sean-brydon — APPROVED** [review]

**Review by PeerRich — CHANGES_REQUESTED** [review]
@Gohan-18 isnt 

`<meta name="viewport" 
      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
`

better?

**Comment by Gohan-18** [comment]
> 

Yeah... That's good too. Let me just modify the code. Thanks.

**Comment by Gohan-18** [comment]
Hey @PeerRich , one small help. Couldn't actually locate the file where to add the tag that works throughout app. Tried in both the theme.config.js and index.html, but doesn't seems to work. Can you please just help in locating the file. Thanks.

**Comment by PeerRich** [comment]
`<meta>` should be in  _document.tsx line 51

**Comment by Gohan-18** [comment]
> `<meta>` should be in _document.tsx line 51

Just updated. Thanks again for helping out

> `<meta>` should be in _document.tsx line 51

Just updated. Thanks again for being so patience and helpful.🙏

File: apps/web/pages/_document.tsx [hunk 1]
```diff
@@ -46,8 +46,8 @@ class MyDocument extends Document<Props> {
           <link rel="manifest" href="/site.webmanifest" />
           <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#000000" />
           <meta name="msapplication-TileColor" content="#ff0000" />
-          <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
-          <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
+          <meta name="theme-color" content="var(--cal-bg)" />
+          <meta name="viewport"  content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"></meta>
```
Comment (PeerRich) [inline apps/web/pages/_document.tsx:50]:
```suggestion
          <meta name="viewport"  content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
```

File: apps/web/styles/globals.css [hunk 2]
```diff
@@ -287,6 +287,7 @@ select:focus {
 
 /* hide chat bubble on mobile */
 @media only screen and (max-width: 768px) {
+
```
Comment (PeerRich) [inline apps/web/styles/globals.css:290]:
```suggestion
```

**Review by PeerRich — APPROVED** [review]

**Review by PeerRich — APPROVED** [review]

File: apps/web/pages/_document.tsx [hunk 3]
```diff
@@ -48,6 +48,7 @@ class MyDocument extends Document<Props> {
           <meta name="msapplication-TileColor" content="#ff0000" />
           <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
           <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
+          <meta name="viewport"  content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```
Comment (PeerRich) [inline apps/web/pages/_document.tsx:51]:
```suggestion
          <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```

File: apps/web/pages/_document.tsx [hunk 4]
```diff
@@ -48,6 +48,7 @@ class MyDocument extends Document<Props> {
           <meta name="msapplication-TileColor" content="#ff0000" />
           <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
           <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
+          <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```
Comment (PeerRich) [inline apps/web/pages/_document.tsx:51]:
```suggestion
          <meta
            name="viewport"
            content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
          />
```

**Pull request merged**